src/Mod/Path/PathScripts/PathDressupHoldingTags.py

```diff
@@ -134,11 +134,10 @@ def __init__(self, nr, x, y, width, height, angle, radius, enabled=True):
         self.height = math.fabs(height)
         self.actualHeight = self.height
         self.angle = math.fabs(angle)
-        self.radius = (
-            radius
-            if FreeCAD.Units.Quantity == type(radius)
-            else FreeCAD.Units.Quantity(radius, FreeCAD.Units.Length)
-        )
+        if hasattr(radius, "Value"):
```

Comment:
I prefer the getattr() method.
The whole if else block can be replaced with one line
https://www.geeksforgeeks.org/python-getattr-method/

Reply:
Just pushed the suggested improvement.